Cargo.toml

```
@@ -130,7 +130,7 @@ serde = { version = "1.0.216", features = ["derive", "rc"] }
 serde_json = { version = "1.0.133", features = ["preserve_order"] }
 similar-asserts = { version = "1.6.0" }
 smallvec = { version = "1.7.0", features = ["union"] }
-stack-graphs = { version = "0.13.0" }
+stack-graphs = { path = "submodules/stack-graphs/stack-graphs", version = "0.14.0" }
```

If we only need the Cargo reference to build this, without any infra/pre-build steps, I wonder why are we adding the submodule alltogether?

We can just add the crate as a direct `git` reference, and it will be cloned/built automatically by `Cargo`?

```suggestion
stack-graphs = { git = "https://github.com/NomicFoundation/stack-graphs", branch = "XXX" }
```

---

Oh, that's a good idea. And it would handle the previous comment as well.